doc/howto/storage_backup.md

```diff
@@ -0,0 +1,109 @@
+
+Use the following command to display the snapshots for a storage volume:
+
+    lxc storage volume info <pool_name> <volume_name>
+
+You can view or modify snapshots in a similar way to storage volumes, by referring to the snapshot with `<volume_name>/<snapshot_name>`.
+
+To show information about a snapshot, use the following command:
+
+    lxc storage volume show <pool_name> <volume_name>/<snapshot_name>
+
+To edit a snapshot (for example, to add a description or change the expiry date), use the following command:
+
+    lxc storage volume edit <pool_name> <volume_name>/<snapshot_name>
+
+To delete a snapshot, use the following command:
+
+    lxc storage volume delete <pool_name> <volume_name>/<snapshot_name>
+
+### Schedule snapshots
+
+You can configure a storage volume to automatically create snapshots at specific times.
+To do so, set the `snapshots.schedule` configuration option for the storage volume (see {ref}`storage-configure-volumes`).
+
+For example, to configure daily snapshots, use the following command:
+
+    lxc storage volume set <pool_name> <volume_name> snapshots.schedule @daily
+
+To configure taking a snapshot every day at 6 am, use the following command:
+
+    lxc storage volume set <pool_name> <volume_name> snapshots.schedule "0 6 * * *"
+
+When scheduling regular snapshots, consider setting an automatic expiry (`snapshots.expiry`) and a naming pattern for snapshots (`snapshots.pattern`).
+See the {ref}`storage-drivers` documentation for more information about those configuration options.
+
+### Restore a snapshot
+
+You can restore a storage volume to the state of any of its snapshots.
+
+To do so, you must first stop all instances that use the storage volume.
+Then use the following command:
+
+    lxc storage volume restore <pool_name> <volume_name> <snapshot_name>
+
+## Use export files for backup
+
+You can export the full content of your storage volume to a standalone file that can be stored at any location.
+For highest reliability, store the backup file on a different file system to ensure that it does not get lost or corrupted.
+
+Use the following command to export a storage volume to a compressed file (for example, `/path/to/my-backup.tgz`):
+
+    lxc storage volume export <pool_name> <volume_name> [<file_path>]
+
+If you do not specify a file path, the export file is saved as `backup.tar.gz` in the working directory.
+
+```{warning}
+If the output file (`backup.tar.gz` or the specified file path) already exists, the command overwrites the existing file without warning.
+```
+
+You can add any of the following flags to the command:
+
+`--compression`
+: By default, the output file uses gzip compression.
+  You can specify a different compression algorithm (for example, `bzip2`) or turn off compression with `--compression=none`.
+
+`--optimized-storage`
+: If your storage pool uses the `btrf` or the `zfs` driver, add the `--optimized-storage` flag to store the data in an optimized way.
```

Comment:
Is it correct that this is only for `btrfs` and `zfs`?
Also, what does "in an optimized way" mean - what is the benefit? I did a few tests, and the size of the optimized export was mostly bigger than the "normal" export.

Reply:
So using `--optimized-storage` would not make sense in combination with `--volume-only`, or if there are no snapshots?